driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -339,6 +370,20 @@ pub mod tests {
 
     #[test]
     fn test_serialize_tokens() {
+        let mut token_data = HashMap::new();
+        let token_info_1 = Some(TokenInfo {
+            alias: String::from("T1"),
+            decimals: 18,
+            external_price: 1_000_000_000_000_000_000,
```

Comment:
Now I see why everybody seems to be fine with this large integers! :P

Reply:
I think python supports number separators as well:
```
Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 1_000_000_000_000
1000000000000
```

Reply:
Oh wow I wasn't aware! Thanks!

Reply:
The only reason I know is that Rust copied its notation from Python :)

Reply:
It would be great if json format would accept that for integers :)